src/OWML.Common/Interfaces/Menus/IOWMLPopupInputMenu.cs

```diff
@@ -4,7 +4,9 @@ namespace OWML.Common.Interfaces.Menus
 {
 	public interface IOWMLPopupInputMenu
 	{
-		public event PopupInputMenu.InputPopupValidateCharEvent OnInputPopupValidateChar;
+		public delegate bool InputPopupValidateCharEvent(string input, int charIndex, char addedChar);
```

Comment:
Would this break mods that use this expecting only a `char` to be passed?

Reply:
yes but considering no mod has used it the event yet I think it is fine